### DISCUSSION TITLE
【2023.4.19】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#2191] [Community meeting]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by kakaZhou719** [comment]
1. sealer v0.10.0 进展同步
2. 开源之夏进展同步
3. plugin 仓库

**Comment by kakaZhou719** [comment]
4月20日 【会议纪要】


1. sealer 0.10.0 版本计划和进度同步。@kakaZhou719
2. sealer 支持oci registry zot，sealer 代码目前完成，待检视。build 带zot registry的基础镜像，需要提供oss地址。@jsparter
3. sealer load更新vendor目录，支持设置buildah的tmpdir,代码目前在开发中。@cubxxw
4. 开源之夏进展同步，目前2个议题已经提交，待审核状态。@kakaZhou719
5. sealer 开源了plugin 仓库，插件内容待丰富。@kakaZhou719
